pkg/doc/did/schema.go

```diff
@@ -22,8 +22,15 @@ const (
         }
       ],
       "additionalItems": {
-        "type": "string",
-        "format": "uri"
+        "oneOf": [
```

Comment:
For verifiable credentials, we have the following schema definition for `@context`:
```
"@context": {
  "oneOf": [
    {
      "type": "string",
      "const": "https://www.w3.org/2018/credentials/v1"
    },
    {
      "type": "array",
      "items": [
        {
          "type": "string",
          "const": "https://www.w3.org/2018/credentials/v1"
        }
      ],
      "uniqueItems": true,
      "additionalItems": {
        "oneOf": [
          {
            "type": "object"
          },
          {
            "type": "string"
          }
        ]
      }
    }
  ]
}
```

There is one test case in VC test suite where the context is defined as follows: 
```
"@context": [
  "https://www.w3.org/2018/credentials/v1",
  "https://www.w3.org/2018/credentials/examples/v1",
  {
    "image": { "@id": "schema:image", "@type": "@id" }
  }
]
```
So: we should support any objects in `@context`

Reply:
Correct - we should be ignoring unknown properties when processing as JSON.

We also need a followup for proper JSON-LD processing as @llorllale mentioned above.

Reply:
done